.github/workflows/build-container.yml

```
@@ -0,0 +1,61 @@
+name: Build container Test
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  build-container-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          # git submodule update --init --recursive
+          submodules: recursive
+
+      - name: Build container images
+        run: make build
+
+      - name: Setup huge pages
+        run: make setup
+
+      - name: Compose Up
+        run: |
+          make up || (make logs; exit 1)
+
+      - name: Wait for controller to start
+        run: |
+          until nc -z localhost 5500; do
+            echo -n .
+            sleep 1
+          done
+          echo
+
+      - name: List containers
+        run: |
+          docker-compose ps
+
+      - name: Create RBD image
+        run: |
+          echo "💁 ceph list pools:"
+          docker exec ceph-cluster ceph osd lspools
+          echo "💁 rbd create:"
+          docker exec ceph-cluster rbd create rbd/mytestdevimage --size 16
+          echo "💁 ls rbd:"
+          docker exec ceph-cluster rbd ls rbd
+
+      - name: Run CLI
+        run: |
+          docker exec ceph-nvmeof_ceph-nvmeof_1 python3 -m control.cli create_bdev -i mytestdevimage -p rbd -b Ceph0
+          docker exec ceph-nvmeof_ceph-nvmeof_1 python3 -m control.cli create_subsystem -n nqn.2016-06.io.spdk:cnode1 -s SPDK00000000000001
+          docker exec ceph-nvmeof_ceph-nvmeof_1 python3 -m control.cli add_namespace -n nqn.2016-06.io.spdk:cnode1 -b Ceph0
+          docker exec ceph-nvmeof_ceph-nvmeof_1 python3 -m control.cli add_host -n nqn.2016-06.io.spdk:cnode1 -t '*'
+          docker exec ceph-nvmeof_ceph-nvmeof_1 python3 -m control.cli create_listener -n nqn.2016-06.io.spdk:cnode1 -s 5001
```

In my original PR #90, I'm creating a separate `ceph-nvmeof-cli` container so that this would run as:

```suggestion
          docker-compose run ceph-nvmeof-cli create_bdev -i mytestdevimage -p rbd -b Ceph0
          docker-compose run ceph-nvmeof-cli create_subsystem -n nqn.2016-06.io.spdk:cnode1 -s SPDK00000000000001
          docker-compose run ceph-nvmeof-cli add_namespace -n nqn.2016-06.io.spdk:cnode1 -b Ceph0
          docker-compose run ceph-nvmeof-cli add_host -n nqn.2016-06.io.spdk:cnode1 -t '*'
          docker-compose run ceph-nvmeof-cli create_listener -n nqn.2016-06.io.spdk:cnode1 -s 5001
```

The main issue with this is that, by running this insider the nvmeof gateway, we're not testing inter-container connectivity, and given we're exposing a gRPC service we should test some networking (in fact, while trying to test this I found that probably we need the `control.cli` module not to use the `ceph-nvmeof.conf`, but specific command line args https://github.com/ceph/ceph-nvmeof/issues/135).

---

Sounds good 🖖, see the following [change](https://github.com/ceph/ceph-nvmeof/pull/134/commits/96e760093d763b4bb74235fc6e461c8685b304c5) to address #135